### PR TITLE
ci: Add semantic release for 1 button release

### PR DESCRIPTION
**Pull request by rmi22186**

### CODE REVIEW

**Comment by mchuangatmp** [comment]
@rmi22186 for kits on android, we're doing more continuous delivery style (so no development branch) because of dependabot (no manual release step to update kits faster)...did you want to go that direction for web?

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+            - name: Install dependencies
+              run: npm ci
+
+            - name: Run tests
+              run: npm run test
+
+            - name: Curl release.config.js and bundle-and-commit.sh
+              run: |
+                  curl "https://raw.githubusercontent.com/rmi22186/workflows/master/web-kits/release.config.js" > release.config.js || exit 1
```
Comment (mchuangatmp) [inline .github/workflows/release.yml]: this needs to be changed to stable tag to merge

Comment (rmi22186) [reply .github/workflows/release.yml]: good catch.  fixed.

**Comment by rmi22186** [comment]
> @rmi22186 for kits on android, we're doing more continuous delivery style (so no development branch) because of dependabot (no manual release step to update kits faster)...did you want to go that direction for web?

@mchuangatmp - per our squad conversation, we will continue to go with the 1 click release for web kits.

**Review by mchuangatmp — APPROVED** [review]

**Pull request merged**